server.py

```diff
@@ -21,6 +21,15 @@
 import threading
 import traceback
 
+## Utilities
+
+def unique(lst):
+    new_lst = []
+    for item in lst :
+        if item not in new_lst :
+            new_lst += [item]
```

Comment:
Use `.append`.

Reply:
Why is append preferred?

Reply:
Seems to me like an abuse of the `+=` operator, like writing `new_lst.extend([item])`. Also, consistency of style.